src/stylesheets/components/_footer.scss

```diff
@@ -272,3 +272,43 @@ li.usa-footer-primary-content {
     padding-left: 1em;
   }
 }
+
+.usa-social-link {
+  $hit-area: 4.5rem; // used for height and width
+  $left-align-adjust: 1.2rem; // relative left positioning
+  $blank-space: 1rem; // spacing between link hit areas
+
+  background-position: center center;
+  background-repeat: no-repeat;
+  display: inline-block;
+  height: $hit-area;
+  left: $left-align-adjust;
+  margin-left: $blank-space;
+  position: relative;
+  text-align: center;
+  width: $hit-area;
```

Comment:
I don't think we need variables. We only use variables if the value is repeated at least twice and the value is likely to be updated at least once.